pkg/plugins/manager/loader/loader.go

```diff
@@ -358,25 +362,21 @@ func setDefaultNavURL(p *plugins.Plugin) {
 	}
 }
 
-func configureAppChildPlugin(parent *plugins.Plugin, child *plugins.Plugin) {
+func configureAppChildPlugin(cfg *config.Cfg, parent *plugins.Plugin, child *plugins.Plugin) {
 	if !parent.IsApp() {
 		return
 	}
-	appSubPath := strings.ReplaceAll(strings.Replace(child.FS.Base(), parent.FS.Base(), "", 1), "\\", "/")
 	child.IncludedInAppID = parent.ID
 	child.BaseURL = parent.BaseURL
 
+	appSubPath := strings.ReplaceAll(strings.Replace(child.FS.Base(), parent.FS.Base(), "", 1), "\\", "/")
 	if parent.IsCorePlugin() {
-		child.Module = util.JoinURLFragments("app/plugins/app/"+parent.ID, appSubPath) + "/module"
+		child.Module = path.Join("app/plugins/app", parent.ID, appSubPath, "module")
```

Comment:
Yeah this one is a weird one - because it's only relevant in a situation where we have a core app plugin, which has a nested plugin 😮 Will definitely double check!

Reply:
cc @jackw let's check this one over

Reply:
>I will test the changes later, but I am flagging this for now in case it's just an oversight:
> Shouldn't it be module.js at the end rather than module?
> Shouldn't this also include the cfg.GrafanaAppSubURL prefix?

Core plugins don't load a `module.js` file in the same way external plugins do. The core plugin module property (that you've highlighted here) is checked [against a map in the frontend](https://github.com/grafana/grafana/blob/jackw/bump-systemjs/public/app/features/plugins/built_in_plugins.ts) and if matched bypasses SystemJS entirely and delivers the module from the grafana webpack bundle. Given we've changed the keys here to `core:plugin/<plugin_name>` we should likely update them here too. To the best of my knowledge we have no core plugins with nested plugins.